<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrimaryHMI</class>
 <widget class="QWidget" name="PrimaryHMI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>697</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PrimaryHMI</string>
  </property>
  <widget class="QLabel" name="WelcomeLabel">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>20</y>
     <width>671</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>17</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>WELCOME TO THE STORE MANAGER</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="MessageLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>691</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Please choose an option</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>551</width>
     <height>129</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="PrimaryMenuLayout">
    <property name="leftMargin">
     <number>15</number>
    </property>
    <property name="topMargin">
     <number>15</number>
    </property>
    <property name="rightMargin">
     <number>15</number>
    </property>
    <property name="bottomMargin">
     <number>15</number>
    </property>
    <property name="spacing">
     <number>15</number>
    </property>
    <item row="0" column="0">
     <widget class="QPushButton" name="CreateBillPushButton">
      <property name="text">
       <string>Create a bill</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="SearchItemPricePushButton">
      <property name="text">
       <string>Search for Item Price</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="SearchItemCodePushButton">
      <property name="text">
       <string>Search for Item Code</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="DisplayInventoryPushButton">
      <property name="text">
       <string>Display All items in inventory</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="AddNewItemPushButton">
      <property name="text">
       <string>Add new item to inventory</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="RemoveInvemtoryItemPushButton">
      <property name="text">
       <string>Remove an item from inventory</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
